app/src/main/java/com/codesoom/assignment/controllers/ProductController.java

```diff
@@ -11,6 +11,9 @@
 import javax.validation.Valid;
 import java.util.List;
 
+/**
+ * 상품의 CRUD 에 대한 http 요청을 처리하는 controller 입니다.
```

Comment:
이 정도로도 충분할 것 같습니다. `CRUD`에 대해서는 책임을 나눌 필요가 있지는 않은지 잘 생각해 보세요.
```suggestion
 * 상품의 CRUD 에 대한 http 요청을 처리합니다.
```